### DISCUSSION TITLE
Swell Attribute Limit

### DISCUSSION THREAD

**Discussion by JanikHalder** [swellstores/community#5] [Help] [answered]
Hi there,

is there an attribute limit in Swell?
Because I need to import over 1.000 Attributes and if I get to 1.001, I always get a 404 Error if I click on an attribute.
Even if I create the 1.001 inside Swell it is not working.

Best,
Janik

**Comment by logeshswell** [comment, marked as answer]
@JanikHalder I guess it's a bug and thanks for the detailed information. Let me raise an internal bug ticket for faster resolution.

**Reply by JanikHalder** [reply]
Thank you @logeshswell.

Can you give me a date, when this issue will be fixed?

Best, Janik
